examples/basic_example/basic_example.py

```diff
@@ -71,28 +71,49 @@
 from oemof.solph import EnergySystem
 from oemof.solph import Model
 from oemof.solph import buses
-from oemof.solph import components as cmp
+from oemof.solph import components as solver_components
```

Comment:
It's actually not the "solver" components. I'd either not give an alias name
```suggestion
from oemof.solph import components
```
or just `import oemof.solph` and explicitly name the submodes in the code.

Reply:
Hi Patrik, thanks for the headsup - I will introduce your suggestion on this